packages/hot-module-replacement/client.js

```diff
@@ -61,7 +60,7 @@ function handleMessage(message) {
       console.log('HMR: Will enable HMR the next time the page is loaded');
       mustReload = true;
     } else {
-      console.log(`HMR: Register failed for unknown reason`, message);
+      console.log('HMR: Register failed for unknown reason', message);
```

Comment:
I think we should start using Logging package for log output.

Reply:
Since the logging package depends on ecmascript, it would create a circular dependency.